mne/preprocessing/interpolate.py

```diff
@@ -153,3 +198,40 @@ def interpolate_bridged_electrodes(inst, bridged_idx):
 
     inst.info['bads'] = bads_orig
     return inst
+
+
+def _find_centroid(ch_pos, group_names):
```

Comment:
Otherwise it seems like a standard centroid
```suggestion
def _find_centroid_sphere(ch_pos, group_names):
```